tests/create_LWE.py

```diff
@@ -0,0 +1,242 @@
+from addict import Dict
+
+from CADETProcess.simulator import Cadet
+
+from CADETProcess.processModel import (
+
+    ComponentSystem,
+    FlowSheet,
+
+    Inlet,
+    Outlet,
+    Cstr,
+    GeneralRateModel,
+    TubularReactor,
+    LumpedRateModelWithoutPores,
+    LumpedRateModelWithPores,
+    MCT,
+
+    Linear,
+    Langmuir,
+    LangmuirLDF,
+    BiLangmuir,
+    BiLangmuirLDF,
+    FreundlichLDF,
+    StericMassAction,
+    AntiLangmuir,
+    Spreading,
+    SelfAssociation,
+    BiStericMassAction,
+    MultistateStericMassAction,
+    SimplifiedMultistateStericMassAction,
+    Saska,
+    GeneralizedIonExchange
+)
+
+
+def create_lwe(n_comp=4, unit_type=None, **kwargs) -> None:
+
+    def configure_all():
+        unit_types = [
+            'cstr', 'generalratemodel', 'tubularreactor',
+            'lumpedratemodelwithoutpores', 'lumpedratemodelwithpores', 'mct'
+        ]
+        for unit_type in unit_types:
+            configure_single_unit(unit_type)
+
+    def configure_single_unit(type_):
+        component_system = ComponentSystem(n_comp)
+        flow_sheet = FlowSheet(component_system)
+
+        if unit_type == 'cstr':
+            configure_cstr(component_system, kwargs)
+        elif unit_type == 'generalratemodel':
+            configure_grm(component_system, kwargs)
+        elif unit_type == 'tubularreactor':
+            configure_tb(component_system, kwargs)
+        elif unit_type == 'lumpedratemodelwithoutpores':
+            configure_lrm(component_system, kwargs)
+        elif unit_type == 'lumpedratemodelwithpores':
+            configure_lrmp(component_system, kwargs)
+        elif unit_type == 'mct':
+            configure_mct(component_system, kwargs)
+        else:
+            raise ValueError(f'Unknown unit operation type {type_}')
+
+    if unit_type is None:
+        configure_all()
+    else:
+        configure_single_unit(unit_type)
+
+
+def configure_cstr(component_system, adsorption_parameters):
```

Comment:
[observation] This function should return the configured `Cstr` object.

Reply:
Yes, all of the methods will eventually do that, but it's just not finished yet...

Reply:
Originally my idea was to configure the process inside the create_lwe method so I wouldn't have to pass down the flow_sheet